src/reference/config/writing-platform-configs.rst

```diff
@@ -60,114 +60,195 @@ What Are Install Targets?
+required files which are on the scheduler host will be accessible on the
+desktop. From Cylc's point of view, the desktop and scheduler hosts are
+considered different platforms but must share an :term:`install target`.
+Cylc needs to be told that these platforms share an install target and so we
+configure this using the designated configuration item:
+:cylc:conf:`global.cylc[platforms][<platform name>]install target`.
 
 .. code-block:: cylc
-   :caption: part of a ``global.cylc`` config file
+   :caption: the ``global.cylc`` config file for this scenario could look like:
 
    [platforms]
-       [[myhost]]
-           hosts = myhost
-           install target = myhost
+       [[localhost]]
+           hosts = localhost
+           install target = localhost
+       [[desktop\d\d\d]]
+           install target = localhost
 
+The ``localhost`` platform is the Cylc Scheduler host, as configured in
+:cylc:conf:`global.cylc[scheduler][run hosts]available`. This is the host that
+the workflow will start on.
+The ``desktop\d\d\d`` platform will set up 1000 platforms, all with the same
+specification and one host per platform.
+
+.. note::
+
+   Cylc carries out a "fullmatch" regular expression comparison with the
+   the platform name so ``desktop\d\d\d`` is effectively the same as
+   ``^desktop\d\d\d$``.
+
+With this config setup, Cylc is aware that these two platforms do not require
+remote installations. Since the job runner is not set, this will default to
+background.
+Cylc has optional configuration ``[[[meta]]]`` to add a description of the
+platform, this may be helpful to use, we will add a platform description to our
+desktop platform.
+In addition, we can take advantage of inbuilt defaults for install target and
+hosts,  resulting in a simplified configuration.
+
+.. code-block:: diff
+
+   [platforms]
+       [[localhost]]
+   -        hosts = localhost
+   -        install target = localhost
+       [[desktop\d\d\d]]
+           install target = localhost
+   +       [[[meta]]]
+   +           description = "Background job on a desktop system"
+
+
+
+If a user wants to run a job on their local desktop, e.g. "desktop123", they should
+set:
+
+   .. code-block:: cylc
+
+      [runtime]
+          [[mytask]]
+              platform = desktop123
+
+in their workflow configuration.
+If ``flow.cylc[runtime][mytask]platform`` is unset, the job will run on the Cylc
+Scheduler host using this default ``localhost`` platform. It may be appropriate
+to allow users to run small jobs on the Cylc Server however more intensive jobs
+should run on a more appropriate platform, as the next scenario will detail.
 
 Cluster with Multiple Login Nodes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- **Platforms with multiple hosts require job runner to be set**
 - **Platforms can group multiple hosts together.**
 
 .. admonition:: Scenario
 
    You have a cluster where users submit to a single Slurm job queue from
```

Comment:
"to a single Slurm job queue" -> "jobs to Slurm"
(Slurm can have multiple queues)